pom.xml

```diff
@@ -87,7 +87,7 @@
             from showing up in the dock and capturing window focus. -->
             -Djava.awt.headless=true
         </argLine>
-        <!-- Our build system (Travis CI) provides a monotonically increasing
+        <!-- Our build system (GitHub Actions) provides a monotonically increasing
         build number. When building locally, this number is obviously absent.
         So we provide a default value. -->
```

Comment:
I'll sync the phrasing with `error-prone-support`, but wonder: does this actually work still? (Something we can test separately; no need to block the PR over it.)